fix(components-core): change TabbedPanels props

scott-williams-az opened the pull request:
### Description

refreshing TabbedPanels with BS5
Removed `app-webdir-ui` logic from `components-core:TabbedPanels`

use `app-webdir-ui/examples/index.html?search-tabs=web_dir_faculty_staff&q=test` to test url parameters
- params should fill in search and select 2nd tab

### Links

- [@asu/components-core](https://unity.web.asu.edu/@asu/components-core/index.html?path=/story/uds-tabbedpanels--default)
- [@asu/app-webdir-ui](https://unity.web.asu.edu/@asu/app-webdir-ui/?path=/story/organisms-search-page-templates--search-page-example)
- [JIRA ticket](https://asudev.jira.com/browse/UDS-1412)
- [Unity Design Kit](https://xd.adobe.com/view/56f6cb78-9af5-4b12-b4ce-ef319f71113f-03a5/screen/674f1d96-f175-442a-8087-25cf3924e219/)

### Checklist

- [x] Unity project successfully builds from root `yarn install` & `yarn build`
- [x] Commits do not contain multiple scopes
- [x] Add/updated examples
- [x] No new console errors
- [x] Accessibility checks

### Browsers

- [x] Chrome
- [x] Safari
- [x] Firefox
- [x] Edge

### Images

<!-- Provide screenshots -->

scott-williams-az committed:
fix(app-degree-pages): Update TabbedPanels

Remove react-router-dom

scott-williams-az committed:
fix(app-webdir-ui): SearchParams props TabbedPanel

SearchParams removed from TabbedPanel
InitialTab required to pass searchParam
us callback onTabChange to update searchParams

scott-williams-az committed:
fix(components-core): update TabbedPanels props

scott-williams-az committed:
fix(components-core): change TabbedPanels props

id prop no longer used, added initialTab prop

BREAKING CHANGE: TabbedPanels Props have changed

davidornelas11 reviewed (commented):
Aside from the error in proptypes on lint, this was tested in a webspark environment using ddev as well as a local ddev isearch environment and everything works!

scott-williams-az committed:
chore(components-core): TabbedPanels pass test

davidornelas11 reviewed (dismissed):
LGTM

scott-williams-az committed:
fix(components-core): change TabbedPanels props

null could mean the content is still loading, remove fallback if selectedTabID is null

mlsamuelson reviewed (changes requested):
Hm. Still experiencing issues:
* In Storybook: "Page Search Example" and "Faculty Rank Web Directory" stories, on a fresh load, the default tab detection/selection doesn't seem to be working and no content displays
* In Storybook: the buttons that navigate to different tabs' content work. Yay.
* In HTML example page (for search): default tab/content is selected and displayed.
* In HTML example page (for search): buttons that navigate to tab content don't work. (I tested in a second browser, in case it was caching. Same.)
* In HTML example page for web-directory: page fails to load. Looks like it's failing to get the token for the service call when the page loads - it seems it's hitting a local url instead of the `searchURL` value.

scott-williams-az committed:
chore(components-core): remove unused variables

scott-williams-az committed:
fix(components-core): activeTab less aggressive

scott-williams-az committed:
fix(components-core): fix TabbedPanel activeTab

scott-williams-az commented:
This is ready for another look
